gdrive/drive_client.py

```diff
@@ -174,9 +178,57 @@ def get_files(filename: str) -> List:
     return results["files"]
 
 
+def get_files_by_drive_id(filename: str, drive_id: str):
+    """
+    Get list of files by filename
+    """
+
+    results = (
+        service.files()
+        .list(
+            q=f"name = '{filename}'",
+            corpora="drive",
+            driveId=drive_id,
+            includeTeamDriveItems=True,
+            supportsTeamDrives=True,
+        )
+        .execute()
+    )
+
+    return results["files"]
+
+
+def get_files_in_folder(id: str) -> List:
+    """
+    Get list of files within a folder by folder ID
+    """
+    files = []
+    page_token = None
+    while True:
+        results = (
+            service.files()
+            .list(
+                q=f"'{id}' in parents and trashed=false",
+                supportsAllDrives=True,
+                includeItemsFromAllDrives=True,
+                pageToken=page_token,
+            )
+            .execute()
+        )
+        files.extend(results.get("files", []))
+        page_token = results.get("nextPageToken")
+        if not page_token:
+            break
+    return files
+
+
 def delete_file(id: str) -> None:
     """
     Delete file by id
     """
 
     service.files().delete(fileId=id, supportsAllDrives=True).execute()
+
+
+def export_to_json(id: str) -> str:
+    return json.loads(service.files().get_media(fileId=id).execute())
```

Comment:
why is the file exported to json? I might be json but also might not be. Should probably just be exported as a string.